code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -159,9 +159,11 @@
 			return
 		else
 			return fork.load_food(src, user)
+
 	if((slices_num <= 0 || !slices_num) || !slice_path)
 		return 0
-	if(W.w_class <= 2 && istype(src, /obj/item/weapon/reagent_containers/food/snacks/sliceable) && W.is_sharp() < 0.8) //Make sure the item is valid to attempt slipping shit into it
+
+	if(W.w_class <= 2 && W.is_sharp() < 0.8) //Make sure the item is valid to attempt slipping shit into it

```

Comment:
&& src.slice_path
Otherwise you can put stuff in food but not be able to cut it out.
Hello gone nukedisk.